WebsiteGreetz/src/test/java/WebsiteTest.java

```diff
@@ -24,56 +22,53 @@ private void logIn(WebDriver driver) throws InterruptedException {
         Thread.sleep(5000);
     }
 
-    private boolean findFavorite(WebDriver driver, int countOfFavorites) {
-        if (driver.findElements(By.className("center full-size")).size() > countOfFavorites)
-            return true;
+    private boolean findFavorite(int countOfFavorites) throws InterruptedException {
+        Thread.sleep(5000);
+        return driver.findElements(By.className("full-size")).size() > countOfFavorites;
+    }
 
-        return false;
+    private boolean find5and90euro() throws InterruptedException{
+        return driver.findElement(By.xpath("//div[@class='price-total' and contains(text(), '5,90')]")).isDisplayed();
     }
 
     @Test
     public void addAndCheckFavorite() throws InterruptedException {
         System.setProperty("webdriver.chrome.driver", "src/main/resources/chromedriver.exe");
         driver = new ChromeDriver();
         logIn(driver);
-
-        //ավելացնենք ֆավորիտ
-
         driver.get("https://www.greetz.nl/ballonnen/denken-aan");
         Thread.sleep(3000);
-
-        WebElement item = driver.findElement(By.xpath("//a[@data-id='1142804226']/.."));
-        WebElement favoriteItem = item.findElement(By.className("b-products-grid__item-action"));
+        WebElement item = driver.findElement(By.xpath("//a[@data-id='1142804226']"));
+        WebElement favoriteItem = item.findElement(By.xpath("//parent::div//child::a[@class='b-products-grid__item-action']"));
         favoriteItem.click();
-
-        //ստուգենք ավելացավ, թե չէ
-
         WebElement sideMenu = driver.findElement(By.xpath("//i[@class='page-header__navigation-item-icon b-icon b-icon-hamburger']"));
         sideMenu.click();
-
         WebElement favorite = driver.findElement(By.xpath("//span[@class='b-list--item-subject' and contains(text(), 'Favorieten')]"));
         favorite.click();
-
-        //int countOfFavorites = 0;
-        //Assert.assertTrue(findFavorite(driver, countOfFavorites));
-
-
-
+        int countOfFavorites = 0;
+        Assert.assertTrue(findFavorite(countOfFavorites));
+        //մեկ ու մեջ ա աշխատելու, մի անգամ նշումա, հաջորդ անգամը ՝ հանում
+        driver.quit();
     }
 
     @Test
-    public void cardPriceX3() throws InterruptedException {
-        //login
+    public void cardPriceX2() throws InterruptedException {
+        //լոգին
         System.setProperty("webdriver.chrome.driver", "src/main/resources/chromedriver.exe");
         driver = new ChromeDriver();
```

Comment:
actually, you do the same steps on 36, 37 and 57, 58 lines, in codding, it's known as code repetition, and you should bypass similar cases as much as possible, I will suggest to organize your driver instantiation inside `logIn` method

Reply:
these parts are from different versions.